lochness/transfer/__init__.py

```diff
@@ -309,10 +312,9 @@ def lochness_to_lochness_transfer_s3(Lochness, general_only: bool = True):
             {source_directory}/ \
             s3://{s3_bucket_name}/{s3_phoenix_root}'
 
-    print('Executing aws s3 sync function')
-    print(re.sub('\s+', ' ', command))
-    print(os.popen(command).read())
-    print('aws rsync completed')
+    logger.debug('Executing aws s3 sync function')
```

Comment:
General question, what are the advantages of using `logger` vs `print`? I know the former gives you module name that you specified up there:

> logger = logging.getLogger(__name__)

And time+line no.

Reply:
In the `sync.py`, there is `--debug` option, which commands to print the lines logged with `logger.debug`. If `--debug` is not used, these `loggr.debug` lines don't not get printed.